charts/v0.3.0/templates/rbac_role.yaml

```diff
@@ -0,0 +1,239 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Please reload this file.  https://github.com/openkruise/kruise/pull/193 updated this file.

Reply:
done